### PR TITLE
Navigation bar update

### PR DESCRIPTION
**Pull request by azanbinzahid**
fixing more bugs

### CODE REVIEW

File: src/main/webapp/js/navigation-loader.js [hunk 1]
```diff
@@ -47,6 +47,11 @@ function addLoginOrLogoutLinkToNavigation() {
 
 function addNaviMenues(){
   const navigationElement = document.getElementById('navigation');
+  if (!navigationElement) {
+    console.warn('Navigation element not found!');
```
Comment (froheel) [inline src/main/webapp/js/navigation-loader.js:51]: What would the difference between console.warn and console.log be ?

Comment (froheel) [reply src/main/webapp/js/navigation-loader.js:51]: Is it just the yellow symbol?

Comment (azanbinzahid) [reply src/main/webapp/js/navigation-loader.js:51]: It will raise warning on console. 
.log will just print

**Review by froheel — APPROVED** [review]
great job!

File: src/main/webapp/js/navigation-loader.js (hunk 1 above)
Comment (froheel) [reply src/main/webapp/js/navigation-loader.js:51]: Thank you so much. Got it !

**Pull request merged**